platform-configs/protectli-v1210.robot

```diff
@@ -58,14 +58,15 @@ ${DEVICE_USB_ROOT_PROMPT}=                          root@3mdeb:/home/ubuntu#
 
 ${DEVICE_WINDOWS_USERNAME}=                         user
 ${DEVICE_WINDOWS_PASSWORD}=                         windows
+${DEVICE_WINDOWS_USER_PROMPT}=                      PS C:\\Users\\user>
 ${DEVICE_UBUNTU_USERNAME}=                          ubuntu
 ${DEVICE_UBUNTU_PASSWORD}=                          ubuntu
 ${DEVICE_UBUNTU_USER_PROMPT}=                       ubuntu@3mdeb:~$
 ${DEVICE_UBUNTU_ROOT_PROMPT}=                       root@3mdeb:/home/ubuntu#
 ${3_MDEB_WIFI_NETWORK}=                             3mdeb_Laboratorium
 
 ${DMIDECODE_SERIAL_NUMBER}=                         N/A
-${DMIDECODE_FIRMWARE_VERSION}=                      Dasharo (coreboot+UEFI) v
+${DMIDECODE_FIRMWARE_VERSION}=                      Dasharo (coreboot+UEFI) v0.9.2
```

Comment:
Not sure what hardcoding the version will accomplish. We want to validate against a binary that will contain given version, not to validate against hardcoded version. What's next? Change the platform config whenever you test a different binary with different version?

Reply:
Right, it's better to check this way.